### PR TITLE
#34 Social impact open knowledge

### LINKED ISSUE #34: 🎨 Create Social Impact open knowledge section
#  🎨 Create Social Impact open knowledge section
---

#### Note: This issue is dependent on #31, please complete that issue first

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user I want a section that describes the organisation open knowledge

### Description

This component displays provided text and images side by side on the social impact section.

Then section has the following :

```
1. An image on the right side
2. A paragraph of text
3. A title

```

Screenshot 2023-02-06 at 10 52 47

Your assignment is to implement the component to appear as above, reuse the component on #31.

Please take note of all the design features (including mobile and rounded corners)

#### Link to full design:
```
https://xd.adobe.com/view/7b72572b-b40a-4618-bfa6-5500ab28f13b-2e21/screen/31438bb0-400a-4da8-a374-c0440535c37c

mobile 
-

```
### Technical Analysis

This component will be created in libs/pages/elewa/social-impact

The component will be called social-impact-open-knowledge

#### Image Url

https://res.cloudinary.com/dyl3rncv3/image/upload/v1675690302/elewa-group-website/Images/unnamed_1_m3dvll.png

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the design above.
2. The component is clean and does not have more than 50 lines
3. The component SCSS uses CSS variables for colour, font, ... Connected to the main style.scss
4. The code is isolated to the libs/page/home package, unless a justified reason is specified.
5. The component is aligned using CSS flex
6. The design is responsive for screen, tablet and phone
7. The image renders fast and has multiple srcsets

### PR DESCRIPTION
**Pull request by Bel-94**
This pull request is part of the work on issue #34 done in collaboration with @pauline123

To achieve this, we needed to:

```
- to re-use the banner component #31
- import the BannerModule so that we can re-use the component
- Make it pixel perfect and mobile friendly
```

Fixes #34 

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# Screenshot
Social impact open knowledge with image on the right
Screenshot from 2023-02-16 23-38-18

Social impact open knowledge mobile version
Screenshot from 2023-02-17 08-25-21

Screenshot from 2023-02-16 23-59-15

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Bel-94** [comment]
Hello @IanOdhiambo9 kindly review my PR #210

File: libs/features/components/cards/src/lib/pages/elewa-group-card/elewa-group-card.component.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<div class="row row-cols-1 row-cols-md-3 g-4">
```
Comment (LemmyMwaura) [inline libs/features/components/cards/src/lib/pages/elewa-group-card/elewa-group-card.component.html:1]: This should not be in a pages folder as it's not a page, also the card is already present. Please sync with the latest changes and work from there.

File: libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html [hunk 2]
```diff
@@ -6,4 +16,5 @@
       <elewa-group-home-jobs-section></elewa-group-home-jobs-section>
     </div>
   </elewa-group-elewa-group-main-page> 
-</div>
\ No newline at end of file
+</div>
+>>>>>>> 1f5124ccd6a27958b4d7e74ec6e9dd4b5b85ce42
```
Comment (LemmyMwaura) [inline libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html:20]: ?? remove

File: libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html [hunk 3]
```diff
@@ -1,3 +1,13 @@
+<<<<<<< HEAD
+<div class="home-page">
+  <elewa-group-header></elewa-group-header>
+
+  <elewa-group-home-hero-section></elewa-group-home-hero-section>
+  
+</div>
+
+<!-- <app-elewa-group-card></app-elewa-group-card> -->
+=======
```
Comment (LemmyMwaura) [inline libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html:10]: remove

File: libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { SocialImpactOpenKnowledgeComponent } from './component/social-impact-open-knowledge/social-impact-open-knowledge.component';
+import { BannersModule } from '@elewa-group/features/components/banners';
+import { SocialImpactPageComponent } from './pages/social-impact-page/social-impact-page.component';
```
Comment (LemmyMwaura) [inline libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts:5]: fix order of imports

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Sync with the latest updates and work from there.

**Comment by IanOdhiambo9** [comment]
Too many files changed for this issue. Sync and resolve conflicts then reopen after changes implemented.

**Pull request closed without merging**